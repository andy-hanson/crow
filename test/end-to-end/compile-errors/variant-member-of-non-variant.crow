main void()
	_ bool = m

m variant-member(bool)
