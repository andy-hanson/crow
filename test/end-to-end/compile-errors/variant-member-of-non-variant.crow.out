exit code: 1
stderr:
test/end-to-end/compile-errors/variant-member-of-non-variant.crow 2:14-2:15 There is no function or variable named 'm'.
test/end-to-end/compile-errors/variant-member-of-non-variant.crow 4:10-4:29 Not a variant: bool
Uncaught exception: Error: Reached compile error
	at main
